README.md

```diff
@@ -6,32 +6,65 @@
 This module is IaC - infrastructure as code which contains a nomad job of [hive](https://hive.apache.org/).
 
 ## Content
-1. [Usage](#usage)
+0. [Prerequisites](#prerequisites)
+1. [Compatibility](#compatibility)
 2. [Requirements](#requirements)
     1. [Required software](#required-software)
-    2. [Providers](#providers)
-3. [Inputs](#inputs)
-4. [Outputs](#outputs)
-5. [Modes](#modes)
-6. [Example usage](#example-usage)
+3. [Usage](#usage)
+   1. [Providers](#providers)
+   2. [Intentions](#intentions)
+4. [Inputs](#inputs)
+5. [Outputs](#outputs)
+6. [Modes](#modes)
+7. [Example](#example)
     1. [Verifying setup](#verifying-setup)
         1. [Data example upload](#data-example-upload)
-7. [License](#license)
+8. [Authors](#authors)
+9. [License](#license)
+10. [References](#references)
 
-## Usage
-The following command will run a instance of hive found in the [example](/example) folder.
-```sh
-make test
-```
+## Prerequisites
+Please follow [this section in original template](https://github.com/fredrikhgrelland/vagrant-hashistack-template#install-prerequisites)
+
+## Compatibility
+|Software|OSS Version|Enterprise Version|
+|:---|:---|:---|
+|Terraform|0.13.1 or newer||
+|Consul|1.8.3 or newer|1.8.3 or newer|
+|Vault|1.5.2.1 or newer|1.5.2.1 or newer|
+|Nomad|0.12.3 or newer|0.12.3 or newer|
 
 ## Requirements
+
 ### Required software
-- [GNU make](https://man7.org/linux/man-pages/man1/make.1.html)
+All software is provided and run with docker.
+See the [Makefile](Makefile) for inspiration.
+
+## Usage
+The following command will run hive in the [example/standalone](example/standalone) folder.
+```sh
+make up
+```
 
 ### Providers
 - [Nomad](https://registry.terraform.io/providers/hashicorp/nomad/latest/docs)
 - [Vault](https://registry.terraform.io/providers/hashicorp/vault/latest/docs)
 
+### Intentions
+Service mesh [communication controlled by intentions](https://learn.hashicorp.com/tutorials/consul/get-started-service-networking#control-communication-with-intentions).
+Intentions are required when [consul acl is enabled and consul_acl_default_policy is deny](https://learn.hashicorp.com/tutorials/consul/access-control-setup-production#enable-acls-on-the-agents).
```

Comment:
```suggestion
Intentions are required when [`consul_acl=true` and `consul_acl_default_policy=deny`](https://learn.hashicorp.com/tutorials/consul/access-control-setup-production#enable-acls-on-the-agents).
```

Reply:
This module can be used without vagrant-hashistack, so I would like to decouple from its context in the documentation. WDYT?

```suggestion
Intentions are required when [consul acl is enabled and default_policy is deny](https://learn.hashicorp.com/tutorials/consul/access-control-setup-production#enable-acls-on-the-agents).
```

Reply:
Oh right, that is a very good point. Didn't think of that at all.